Shoko.Server/Databases/MySQL.cs

```diff
@@ -723,7 +723,7 @@ public class MySQL : BaseDatabase<MySqlConnection>
         new(109, 1, "UPDATE VideoLocal v INNER JOIN CrossRef_File_Episode CRFE on v.Hash = CRFE.Hash SET DateTimeImported = DateTimeCreated;"),
         new(110, 1, "CREATE TABLE `AniDB_FileUpdate` ( `AniDB_FileUpdateID` INT NOT NULL AUTO_INCREMENT, `FileSize` BIGINT NOT NULL, `Hash` varchar(50) NOT NULL, `HasResponse` BIT NOT NULL, `UpdatedAt` datetime NOT NULL, PRIMARY KEY (`AniDB_FileUpdateID`) );"),
         new(110, 2, "ALTER TABLE `AniDB_FileUpdate` ADD INDEX `IX_AniDB_FileUpdate` (`FileSize` ASC, `Hash` ASC) ;"),
-        new(110, 3, DatabaseFixes.MigrateAniDB_FileUpdates),
+        new(110, 3, DatabaseFixes.NoOperation),
```

Comment:
What's the reason for killing an existing migration?

Reply:
It is no longer relevant to run since they're later removed, and the models the migration used is no longer available.